src/workers/postTranslated.ts

```diff
@@ -22,11 +22,15 @@ export const postTranslated: TypedWorker<'kvasir.v1.post-translated'> = {
         .set({
           translation: () => `jsonb_set(
           translation,
-          '{${language}}',
-          '${JSON.stringify(translations)}'::jsonb,
+          :language,
+          :translations::jsonb,
           true
         )`,
         })
+        .setParameters({
+          language: [language],
```

Comment:
Just put language in escape as well for good measure

Reply:
yeah, always better then string concat